examples/all-clusters-app/linux/AllClustersCommandDelegate.cpp

```diff
@@ -373,6 +374,49 @@ void EmitOccupancyChangedEvent(EndpointId endpointId, uint8_t occupancyValue)
     }
 }
 
+/**
+ * Named pipe handler for simulating a configuration change
+ * by changing the LevelStep value in the ValveConfigurationAndControl cluster
+ * and incrementing the ConfigurationVersion
+ *
+ * Usage example:
+ *   echo '{"Name":"SimulateConfigurationChange"}' > /tmp/chip_all_clusters_fifo_53713
+ */
+
+void HandleSimulateConfigurationChange(void)
+{
+    EndpointId endpoint = 1;
+
+    // Change a F attribute to simulate a change in configuration of the device
+    uint8_t valveLevelStep = 0;
+    Protocols::InteractionModel::Status status =
+        ValveConfigurationAndControl::Attributes::LevelStep::Get(endpoint, &valveLevelStep);
+    VerifyOrDie(status == Protocols::InteractionModel::Status::Success);
+
+    if (valveLevelStep == 1)
+    {
+        // Change fixed LevelStep value to 10
+        valveLevelStep = 10;
+    }
+    else
+    {
+        // Change fixed LevelStep value back to 1
+        valveLevelStep = 1;
+    }
+
+    status = ValveConfigurationAndControl::Attributes::LevelStep::Set(endpoint, valveLevelStep);
+    if (status != Protocols::InteractionModel::Status::Success)
+    {
+        ChipLogError(NotSpecified, "Failed to set LevelStep value");
+    }
+    else
+    {
+        // LevelStep in ValveConfigurationAndControl has been modified, so bump ConfigurationVersion
+        DataModel::ProviderMetadataTree::ScopedConfigurationVersionUpdater configurationVersionTransaction(
+            InteractionModelEngine::GetInstance()->GetDataModelProvider());
```

Comment:
This basically boils down to a forced bump rather  than scoping all the changes.  Suggest scoping the entire Set of fixed attribute in the scope that has this at the top.